lib/client/db/dbcmd/dbcmd.go

```diff
@@ -582,6 +600,20 @@ func WithNoTLS() ConnectCommandFunc {
 	}
 }
 
+// WithRandomPassword is the connect command option that makes the command
+// generate a random password for the database user.
+func WithRandomPassword() ConnectCommandFunc {
+	return WithPassword(uuid.New().String())
```

Comment:
Right now we the `WithRandomPassword` is used to provide a password placeholder when we are connecting to AWS instance the purpose of WithRandomPassword  is to generate password arg  to bypass `cqlsh`  cli password prompt there the value of this password is never actually used. 
I have switched to ` utils.CryptoRandomHex(16)` 
```
	if database.GetProtocol() == defaults.ProtocolCassandra && database.IsAWSHosted() {
		// Cassandra client always prompt for password, so we need to provide it
		// Provide an auto generated random password to skip the prompt in case of
		// connection to AWS hosted Cassandra.
		password, err := utils.CryptoRandomHex(16)
		if err != nil {
			return trace.Wrap(err)
		}
		opts = append(opts, dbcmd.WithPassword(password))
	}